pkg/crc/config/viper_config_test.go

```diff
@@ -181,13 +181,13 @@ func TestViperConfigBindFlagSet(t *testing.T) {
 	assert.NoError(t, err)
 
 	assert.Equal(t, SettingValue{
-		Value:     6,
+		Value:     5, // flag has the priority
```

Comment:
@guillaumerose does that mean we were ignoring the flag priority over what present in config?

Reply:
no this is a weird case where someone would launch `crc config set cpus 8 --cpus 10`.  This is purely theoretical. Set a key and at the same time set it with flag.

Reply:
@guillaumerose for me, it leads to following error which make sense.
 
```
$ ./crc config set cpus 6 --cpus 10
unknown flag: --cpus
```

But when I am looking the test case I think we are adding the flags and only for test case it might be the case. Can't we just remove it? It will be confusing over time.

Reply:
I removed the assertion :+1: